lms/envs/production.py

```diff
@@ -1133,3 +1134,7 @@ def get_env_setting(setting):
 
 ############## NOTIFICATIONS EXPIRY ##############
 NOTIFICATIONS_EXPIRY = ENV_TOKENS.get('NOTIFICATIONS_EXPIRY', NOTIFICATIONS_EXPIRY)
+
+############## Event bus producer ##############
+EVENT_BUS_PRODUCER_CONFIG = merge_producer_configs(EVENT_BUS_PRODUCER_CONFIG,
```

Comment:
Does this need to be added to the keys-with-merged-values list?

Reply:
and that's what i get for not testing on the LMS. Yes, yes it does.

Reply:
[fixed]